package.json

```diff
@@ -111,7 +111,7 @@
     "react-native-codegen": "0.0.7",
     "react-native-dash": "0.0.11",
     "react-native-dotenv": "^2.5.1",
-    "react-native-linear-gradient": "^2.5.6",
+    "react-native-linear-gradient": "https://github.com/react-native-linear-gradient/react-native-linear-gradient#0e3af58",
```

Comment:
Had to reference this commit directly as they've made a fix to be compatible with `use_frameworks!` but haven't released for months!

Reply:
If this is the case we will need to do some work in the app, for building this specific version. There is some specific artifacts we build for Android manually and publish that we do against this version